core/web/resolver/job.go

```diff
@@ -0,0 +1,137 @@
+package resolver
+
+import (
+	"database/sql"
+	"errors"
+	"strconv"
+
+	"github.com/graph-gophers/graphql-go"
+
+	"github.com/smartcontractkit/chainlink/core/services/job"
+)
+
+// JobResolver resolves the Job type.
+type JobResolver struct {
+	j job.Job
+}
+
+func NewJob(j job.Job) *JobResolver {
+	return &JobResolver{j: j}
```

Comment:
I pretty much always use named fields for readability purposes, otherwise you have to reference the struct for what the first field is, and if it ever gets re-ordered, you need to fix these up.